UM-USElab/gradecraft-development
# Student dashboard graded assignments and ungraded submissions table

### Status
**READY**

### Description
This PR adds three tables to the bottom of the student dashboard page. These tables are very similar to the tables professors see when they look at a student (ex: 'students/9'), with a few minor changes. 

Things to discuss would be the content fo the tables like columns 

note - I chose to use another dynatable rather than try to work through the angular problems associated with modifying the grading status table that is used to populate the 'ungraded submissions' table for professors. 

### Todos
- [x] Converse about table content / language 
- [x] Test in staging
- [x] Check that the table does not appear if there are no items in it

### Migrations
NO

### Steps to Test or Reproduce
Log in as a student and view their dashboard. Now they have two tables at the bottom of this page. 

Tables to compare to this are on the professors 'grading status' page, and a professors view of a student ex: '.../students/6'. These tables are professor specific because they contain information regarding the whole class and grades that may not yet be visible to students. 

If you preview GC as a student, you see the same tables as the student.

======================
Closes #4194 #3454
